templates/_base/base.html

```diff
@@ -33,10 +54,6 @@
   {% endif %}
 
 
-  <!--[if IE]>
-    <meta http-equiv="X-UA-Compatible" content="IE=8">
-  <![endif]-->
-
   <link rel="shortcut icon" href="{{ versioned_static( 'files/favicon.ico') }}" type="image/x-icon" />
   <link rel="apple-touch-icon" sizes="144x144" href="https://res.cloudinary.com/canonical/image/fetch/q_auto,f_auto/https://assets.ubuntu.com/v1/9864667f-apple-touch-icon-144x144-precomposed.png">
```

Comment:
The modern way to include Apple Touch icons is to use a single <link> to a `180px x 180px` icon like so;

`<link rel="apple-touch-icon" href="/example.png">`